src/components/sortable.jsx

```diff
@@ -255,7 +255,7 @@ var Sortable = React.createClass({
         layout: React.PropTypes.oneOf([HORIZONTAL, VERTICAL]),
         padding: React.PropTypes.bool,
         disabled: React.PropTypes.bool,
-        constraints: React.PropTypes.object,
+        constraints: React.PropTypes.shape({}),
```

Comment:
It's still on the react docs and if so, `React.PropTypes.object` specifies it's an object with unspecified keys, which is more specific than `any`, and should allow at least `undefined` unless we use `.isRequired` (i'm not sure if react still counts `null` as valid for non `.isRequired` propTypes).